source/postprocess/dynamic_topography.cc

```diff
@@ -41,7 +41,8 @@ namespace aspect
                                quadrature_formula,
                                update_values |
                                update_gradients |
-                               update_q_points);
+                               update_q_points |
+                               update_JxW_values);
```

Comment:
I see. Awkward. But not worth dealing with here -- your changes are definitely correct one way or the other.

I've opened https://github.com/dealii/dealii/issues/3226